recipes/TIES20/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "TIES20" %}
+{% set version = "20.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/UCL-CCS/TIES/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c0f2acdf4dade70637c505298c5030554c3b689f5858f610da1df708ab39b2c7
+
+build:
+  number: 0
+  skip: True  # [win or osx and py==310]
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - ties = ties:cli.command_line_script
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - numpy
+    - cython
+    - setuptools
+  run:
+    - python
+    - ambertools >=20.0,<=22.0
+    - ties_md =1.4.1
+    - numpy
+    - matplotlib-base
+    - networkx
+    - dwave-networkx
+    - dimod
+    - tabulate
+
+test:
+  imports:
+    - ties
+  requires:
+    - pip
+    - pytest
+  commands:
+    - pip check
+    - ties --help
+    - python -m ties.testing.run
```

Comment:
Thanks for your input it is very helpful. Please may I ask, if we don't run the tests in the site-packages directory what is the alternative location for best practice?

Reply:
On more reflection I think I understand that `python -m ties.testing.run` is a much nicer way to run the tests. It should work provided the test paths are correct which they are since we copy to a tmp dir. Therefore I think things are okay as they are? Please correct me if I'm wrong.

Reply:
[Your tests are failing](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=612349&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=7563), but you are returning exit code 0, so conda doesn't know they are failing.

If the tests don't require generating any new files, then it's not necessary to copy all the tests to a new directory. Just run them in place. Use [importlib](https://docs.python.org/3/library/importlib.resources.html) if you need to access files or maybe `os.path.dir(__file__)`.

Reply:
The tests have been addressed now.